build.sbt

```diff
@@ -8,9 +8,9 @@ lazy val buildSettings = Seq(
 )
 
 lazy val twitterVersion = "19.1.0"
-lazy val circeVersion = "0.11.0"
+lazy val circeVersion = "0.11.1"
 lazy val circeIterateeVersion = "0.12.0"
-lazy val circeFs2Version = "0.11.0"
+lazy val circeFs2Version = "0.11.1"
```

Comment:
@scala-steward I don't think this version has to be updated too.

/cc @fthomas

Reply:
This is unfortunate, but I don't see an easy way to fix this. Currently it is hard for scala-steward to differentiate between dependencies with similar names and the same version numbers but different release cycles.